omniledger/service/api.go

```diff
@@ -227,6 +239,8 @@ type Config struct {
 	Roster onet.Roster
 	// OwnerID is the identity that can sign evolutions of the genesis Darc.
 	OwnerID darc.Identity
+	// GenesisDarc is the first darc stored in omniledger.
+	GenesisDarc darc.Darc
```

Comment:
I'm wondering if this should be a darc ID, because the actual darc might change. But I don't know how this is going to be used, I guess it's in one of your other PRs.

Reply:
It's used later in the `pop`-app, where it stores this configuration and then uses it to communicate with omniledger. It saves two requests to get the config- and then the darc of the config. But the information might be stale.